src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -58,13 +60,182 @@ function argsToSDL(args: GraphQLFieldConfigArgumentMap) {
   return result
 }
 
+const gravityMarketingCollectionsExtensions = gql`
+  extend type Artist {
+    marketingCollections(
+      slugs: [String!]
+      category: String
+      size: Int
+      isFeaturedArtistContent: Boolean
+    ): [MarketingCollection]
+  }
+  extend type Fair {
+    marketingCollections(size: Int): [MarketingCollection]!
+  }
+  extend type Viewer {
+    marketingCollections(
+      slugs: [String!]
+      category: String
+      size: Int
+      isFeaturedArtistContent: Boolean
+      artistID: String
+    ): [MarketingCollection]
+  }
+  type HomePageMarketingCollectionsModule {
+    results: [MarketingCollection]!
+  }
+  extend type HomePage {
+    marketingCollectionsModule: HomePageMarketingCollectionsModule
+  }
```

Comment:
@mdole I think this [resolver](https://github.com/artsy/metaphysics/blob/main/src/lib/stitching/kaws/v2/stitching.ts#L98) (with `{}`) is required because we use a query like this (with `results` nested):

```
homePage {
	marketingCollectionsModule {
		results {
			slug
		}
	}
}
``` 

<img width="668" alt="Screenshot 2022-04-12 at 09 21 12" src="https://user-images.githubusercontent.com/3934579/162903672-c0aec264-dec6-4222-b8b6-2b82264d056b.png">

Reply:
oh yep you're totally right! that's confusing hah 🤔